CHANGELOG.md

```diff
@@ -3,7 +3,25 @@
 This file documents all notable changes in `LogDNA Node.js Code Library`. The release numbering uses [semantic versioning](http://semver.org).
 
 ## Next Release [MAJOR]
```

Comment:
`CHANGELOG.md` should be updated in the process of releasing; so, adding example for the usage `shimProperties` into the `README.md` is enough